src/pages/SignUp/SignUp.tsx

```diff
@@ -4,7 +4,7 @@ import { Box, Grid, Container, Typography } from '@mui/material'
 import { registerUser } from '@/features/auth/authSlice'
 import { useAppDispatch, useAppSelector } from '@/app/hooks'
 import Login from '@/pages/Login'
-import AuthButton from '../AuthButton/AuthButton'
+import AuthButton from '../../components/AuthButton/AuthButton'
```

Comment:
please use @/components...

and resolve conflict

Reply:
Thank you, I will change it.